### PR TITLE
Bump Bazel version to 3.4.1

### PR DESCRIPTION
**Pull request by ImJasonH**
This is the version currently installed by, for example, `brew install bazel`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ install:
   - export PATH=$PATH:$HOME/bin && mkdir -p $HOME/bin
   # install bazelisk as bazel to install the appropriate bazel version
   - wget https://github.com/bazelbuild/bazelisk/releases/download/v1.5.0/bazelisk-linux-amd64 && chmod +x bazelisk-linux-amd64 && mv bazelisk-linux-amd64 $HOME/bin/bazel
```
Comment (loosebazooka) [inline .travis.yml:8]: move bazelisk to 1.6.1 too?

**Comment by chanseokoh** [comment]
The build is failing :(

**Comment by ImJasonH** [comment]
> The build is failing :(

Yeah, looks like even though [Bazel has a 3.4.1](https://github.com/bazelbuild/bazel/releases/tag/3.4.1), [buildifier only has 3.4.0](https://github.com/bazelbuild/buildtools/releases/tag/3.4.0). Both have a recent 3.5.0, within the last week, but it looks like those aren't yet available via homebrew.

**Comment by chanseokoh** [comment]
So what next? Use 3.4.0 or wait for 3.5.0?

**Comment by ImJasonH** [comment]
> So what next? Use 3.4.0 or wait for 3.5.0?

I'm fine with either, and since I just downgraded my local installation to 3.2.0 I'm also fine with just dropping this whole PR. 🤷‍♂️

**Comment by mattmoor** [comment]
I have a working bump here: https://github.com/GoogleContainerTools/distroless/pull/599

Gonna close this out, sorry for cross the streams Jason 😅

**Pull request closed without merging**